zubbi/scraper/repo_parser.py

```diff
@@ -100,6 +103,9 @@ def parse_job_definitions(self, file_path, job_info):
                             exc,
                         )
 
+                # If the repo is configured as reusable, all jobs are considered reusable
+                job.reusable = self.is_reusable_repo or getattr(job, "reusable", False)
```

Comment:
Why do we need to use `getattr()` here?

Reply:
But the attribute should be defined on the model class (e.g. here https://github.com/bmwcarit/zubbi/blob/master/zubbi/models.py#L91). If that's not the case then we have to fix this.

Reply:
Will fix by defining the attribute in models.py.